src/core/main.go

```diff
@@ -18,6 +18,7 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/goharbor/harbor/src/controller/systemartifact"
```

Comment:
please reorg the import lines to make github.com in a group.

Reply:
Done